src/react/src/cardset-preview-tile/CardSetPreviewTile.jsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import PropTypes from 'prop-types';
+import Grid from '@mui/material/Grid';
+import { CardActionArea, CardContent, CardMedia } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import './CardSetPreviewTile.css';
+
+function CardSetPreviewTile({
+  name, tags, id, setClickHandler, currentSetId, previewImageUrl,
+}) {
+  const fallbackImageUrl = 'https://lightwidget.com/wp-content/uploads/local-file-not-found-480x488.png';
+
```

Comment:
gibts da vielleicht ein schöneres bild?
Und würde es nicht sinn machen das in unsere assets zu legen. Wäre schlecht wenn unser fallback image nicht verfügbar ist

Reply:
Das sollte erstmal nur ein Filler sein. Können das gerne jederzeit austauschen.

Reply:
Fallback image wurde durch icon ersetzt.